docs/style_guide.md

```diff
@@ -123,3 +121,27 @@ if (length(foo()) == 1) {
 3. Avoid inline CSS styles.
 4. In CSS file, each rule should be on its own line.
 5. Avoid using percentages for widths; instead, use bootstrap's grid system by specifying columns.
+
+## General package code
+1. Create file _{pkgname}-package.R_ with `usethis::use_package_doc()`
+2. Update _{pkgname}-package.R_ - it should have such lines:
+```
+#' @note To learn more about functions start with `help(package = "{pkgname}")` 
+#' @keywords internal
+#' @return package help page
+"_PACKAGE"
+```
+3. Add in the DESCRIPTION `Roxygen: list(markdown = TRUE)`
+4. All constants, imports and side-effects tools should be in file `package.R`. Do not use _zzz.R_
```

Comment:
To be internally coherent I suggest changing `package.R` to _package.R_

Reply:
👍 good point